docs/src/main/asciidoc/_elytron/components/Filesystem_Security_Realm.adoc

```diff
@@ -13,6 +13,8 @@ Every identity is stored in an XML file where the name of the file is the name o
 * `encoded` When encoding is set to true, the identity names will be BASE32 encoded before they are used as filenames. This is beneficial, because some filesystems are case-insensitive or might restrict set of characters allowed in a filename. Default value is true.
 * `hash-encoding` specifies the string format for the password if it is not stored in plain text. Set to BASE64 bt default, but HEX is also supported.
 * `hash-charset` specifies the character set to use when converting the password string to a byte array. Set to UTF-8 by default.
+* `credential-store` specifies the credential store where the secret key is stored if the filesystem realm is to be encrypted. This attribute is optional.
+* `secret-key` specifies the alias of the secret key that is stored in the previously specified credential store. This attribute is optional. Set to `key` by default.
```

Comment:
The `filesystem-realm` resource doesn't set a default value for `secret-key` so this last part can be removed.